package.json

```diff
@@ -24,6 +24,8 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "babel-middleware": "git://github.com/shane-tomlinson/babel-middleware.git#45c06180",
```

Comment:
@shane-tomlinson seems like you had do a lot of fixes and the module is not maintained that much anymore, is it possible to republish it with a different name and give the rest of the team access to the fork / npm publishing?